tests/op/extract/split/test_markdown_header_splitter.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+
+from uniflow.node import Node
+from uniflow.op.extract.split.markdown_header_splitter import MarkdownHeaderSplitter
+
+
+class TestMarkdownHeaderSplitter(unittest.TestCase):
+    def setUp(self):
+        self.splitter = MarkdownHeaderSplitter("test_splitter")
+
+    def test_call(self):
```

Comment:
provide a meaningful function name

Reply:
It is testing the special method `def __call__(self, x)`. Should I just rename it to test___call__() or test_speical_call()?

Reply:
maybe call it test_callable_object_construction